wgsl/index.bs

```diff
@@ -18767,7 +18767,27 @@ All synchronization functions [=shader-creation error|must=] only be invoked in
     <td>`T` is a [=type/concrete=] [=constructible=] type.
   <tr>
     <td>Description
-    <td>Returns the value pointed to by `p` to all invocations in the workgroup.
+    <td>Returns the non-atomic value pointed to by `p` to all invocations in the workgroup.
+    The return value is [=uniform value|uniform=].
+    `p` [=shader-creation error|must=] be a [=uniform value=].
+
+    Executes a [=control barrier=] synchronization function that affects
+    memory and atomic operations in the [=address spaces/workgroup=] address
+    space.
+</table>
+<table class='data builtin'>
+  <tr>
+    <td style="width:10%">Overload
+    <td class="nowrap">
+      <xmp highlight=wgsl>
+        @must_use fn workgroupUniformLoad(p : ptr<workgroup, atomic<T>, read_write>) -> T
+      </xmp>
+  <tr>
+    <td>Parameterization
+    <td>`T` is a [=type/concrete=] [=integer scalar=] type.
```

Comment:
I have a slight preference to dropping this since `atomic<T>` already implies this restriction and saves a potentially missed spec change if non-integral atomics ever get added.
```suggestion
```

Reply:
Done